sorter.cpp

```diff
@@ -28,10 +28,10 @@ namespace sorter {
             }
         }
 
-        swap(&s[0][0], &s[max_pos][0]);
-        swap(&s[1][0], &s[max2_pos][0]);
-        swap(&s[0][1], &s[max_pos][1]);
-        swap(&s[1][1], &s[max2_pos][1]);
+        s[0][0] = max;
```

Comment:
Well if we don't need to modify the array, why not return just the 2 values? In a struct or something?